.env.development

```diff
@@ -20,3 +20,4 @@ MFE_CONFIG_API_URL=''
 ADDITIONAL_METADATA_REQUIRED_FIELDS='{}'
 IS_NEW_SLUG_FORMAT_ENABLED='false'
 MARKETING_SITE_PREVIEW_URL_ROOT=''
+DEFAULT_EXTERNAL_PRODUCT_SOURCE=''
```

Comment:
nit: `default` word should be omitted, may be we can use `2U` word in this variable.

Reply:
There is no default external source. It should not be here in the first place as it can set misleading expectations.